recipes/linalg/24.11/gh200/config.yaml

```diff
@@ -1,6 +1,6 @@
 name: linalg
 spack:
-  commit: releases/v0.23
+  commit: develop-2025-02-09
```

Comment:
Update to release.

This is not super urgent for me, so this could potentially wait for spack 1.0, but I could also pull in updated dla-future/pika definitions to the recipe here.

Reply:
I bundled newer versions of dla-future, dla-future-fortran, and pika, and reverted this changed.